src/styles/product.scss

```diff
@@ -0,0 +1,144 @@
+.product {
+  background-color: $page-backgroun-color;
+  overflow: scroll;
+
+  &__top {
+    padding: 15px 0 20px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__title {
```

Comment:
You didn't specify the font-family in the &__title block. Check if it is set in a higher-level block or set it explicitly if necessary.